hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -410,11 +413,13 @@
 
     // ext is to create port in the top.
     'external': {
-        'clkmgr.clk_main': 'clk_main',
-        'clkmgr.clk_io': 'clk_io',
-        'clkmgr.clk_usb': 'clk_usb',
-        'clkmgr.clk_aon': 'clk_aon'
-        'rstmgr.ast': '',
+        'clkmgr.clk_main': 'clk_main',   // clock inputs
+        'clkmgr.clk_io': 'clk_io',       // clock inputs
+        'clkmgr.clk_usb': 'clk_usb',     // clock inputs
+        'clkmgr.clk_aon': 'clk_aon',     // clock inputs
+        'clkmgr.clocks_ast': 'clks_ast', // clocks output to ast
+        'rstmgr.ast': '',                // ast reset input
+        'rstmgr.resets_ast': 'rsts_ast', // resets output to ast
```

Comment:
maybe i should auto-generate these connections...

Reply:
if that is not too difficult to add, this would probably makes sense, given that the rst and clk mgr outputs are also autogenerated based on the hjson...

Reply:
added! please take a look.